### ISSUE TITLE
Bublum - Traffic Rules Violation Detection

### ISSUE THREAD

**Issue by shibashism** [hackferenceindia/submissions#10]
## ℹ️ Project information
_Please complete all applicable._

- **Project Name**: Traffic Rules Violation Detection
- **Short Project Description**: Automatic detection of vehicles violating traffic rules along with vehicle monitoring using Tensorflow Object Detection API.
- **Team Name**: Bublum
- **Team Members**: - Rahul Shah [@r-sniper](https://github.com/r-sniper/)
				   - Akshay Kumar [@akzarma](https://github.com/akzarma/)
				   - Shibashis Mallik [@shibashism](https://github.com/shibashism/)
- **Demo Link**: [Demo](https://drive.google.com/file/d/1G3VS-X7uLVO8eGHVrbjyVhpW2RESKTzt/view?usp=sharing) showing the detection of vehicles at a square in Pune.
- **Repository Link**: https://github.com/akzarma/Bublum
- **Labels**: Open Innovation No Track Followed

## 🔥 Your Pitch
You may have seen numerous cameras installed across the city. But currently these cameras are being manually operated by traffic officials. This is very much inefficient in terms of manpower used and also the number of traffic rule violators caught.
To solve this problem we propose this automatic traffic rule violation detector. Our system uses neural network developed with Tensorflow Object Detection API to automatically detect vehicles. These vehicles are then passed to another model which detects traffic rule violation such as not wearing helmets. These vehicles are then passed through another model which recognises the number plate and stores them in the database for further action.
This model can also be used for vehicle monitoring where each vehicle spotted in camera are being stored in the database. This can be used to track lost vehicles or suspicious vehicles.
This would greatly decrease the number of traffic violations thereby increasing public safety. Overall revenue generated also would increase drastically through this approach.

## 🔦 Any other specific thing you want to highlight?
This model can be further implemented for more violations detection such as
- Detecting fancy number plates
- Vehicle on Zebra Crossing
- Vehicles in No Parking Zone

## ✅ Checklist

**Before you post the issue**:
- [x] You have followed the issue title format.
- [x] You have mentioned the correct labels.
- [x] You have provided all the information correctly.
